qemu/tests/cfg/fio_perf.cfg

```diff
@@ -16,29 +16,24 @@
         order_list = "Block_size Iodepth Threads BW(MB/S) IOPS Latency(ms) Host_CPU BW/CPU KVM_Exits Util%"
         guest_ver_cmd = "uname -r"
         check_install_fio = "which fio"
-        tarball = "performance/fio-3.27.tar.gz"
-        fio_path = "/tmp/fio-3.27"
-        compile_cmd = "make && make install"
         pre_cmd = "i=`/bin/ls /dev/[vs]db` && mkfs.xfs $i > /dev/null; partprobe; umount /mnt; mount $i /mnt"
         ppc64, ppc64le:
             virtio_blk:
                 pre_cmd = "i=`/bin/ls /dev/vda` && mkfs.xfs $i > /dev/null; partprobe; umount /mnt; mount $i /mnt"
         drop_cache = "sync && echo 3 > /proc/sys/vm/drop_caches"
         guest_result_file = /tmp/fio_result
-        fio_cmd = "fio --rw=%s --bs=%s --iodepth=%s --runtime=1m --direct=1 --filename=/mnt/%s --name=job1 --ioengine=libaio --thread --group_reporting --numjobs=%s --size=512MB --time_based --output=/tmp/fio_result &> /dev/null"
+        fio_options = "--rw=%s --bs=%s --iodepth=%s --runtime=1m --direct=1 --filename=/mnt/%s --name=job1 --ioengine=libaio --thread --group_reporting --numjobs=%s --size=512MB --time_based --output=/tmp/fio_result &> /dev/null"
     Windows:
         virtio_blk:
             guest_ver_cmd = wmic datafile where name="c:\\windows\\system32\\drivers\\viostor.sys"
         virtio_scsi:
             guest_ver_cmd = wmic datafile where name="c:\\windows\\system32\\drivers\\vioscsi.sys"
         order_list = "Block_size Iodepth Threads BW(MB/S) IOPS Latency(ms) Host_CPU BW/CPU KVM_Exits"
         check_install_fio = "dir C:\fio-3.9-x64\ | findstr /I fio"
-        tarball = "performance/fio-3.9-x64/*"
-        fio_path = "C:\fio-3.9-x64"
```

Comment:
For windows guest, with out `fio_ver` , it will use the default one:
https://github.com/autotest/tp-qemu/blob/d074ae9577478a0d056ce83314c7d473bb9ff6da/provider/storage_benchmark.py#L373

It is not match with the current version for you test, please check which one you want to use.

Reply:
And, please note, the version info be hard code to the `check_install_fio` too.

Reply:
@hellohellenmao  Just double confirm,  are you use `fio-3.1` and drop `fio-3.9` on purpose ? 
It is better to use the same version as before, but i am ok if it is intentional from feature testing view. Thanks!

Reply:
Yes, Yanan. It's okay with the version of fio-3.1.